src/components/Dashboard/NewsItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Popup from "reactjs-popup";
+
+const NewsItem = ({ post }) => {
+	const options = { year: 'numeric', month: 'long', day: 'numeric' };
+	return (
+		<tr className="row news-item">
+			<td className="news-item__info">
+				<div className="news-item__info__title">
+					<a href={post.link}>{post.title.rendered}</a>
+				</div>
+				<div className="news-item__info__meta">
+					<span className="news-item__info__meta__modified">{new Date(post.modified).toLocaleDateString( 'en-GB', options )}</span>
```

Comment:
Would be cool to add the locale to the JS `HM` var to pass in here. May need to replace the underscore with a hyphen though.